cedar-lean/Cedar/Thm/Partial/Authorization/PartialOnConcrete.lean

```diff
@@ -200,4 +200,53 @@ theorem knownForbids_eq_forbids {policies : Policies} {req : Spec.Request} {enti
   unfold Partial.Response.knownForbids Partial.Response.forbids
   apply mustBeSatisfied_eq_mayBeSatisfied (eff := .forbid) wf
 
+/--
+  on concrete inputs, `Partial.Response.errors` and `Spec.errorPolicies` are equal
+-/
+theorem errors_eq_errorPolicies {policies : Policies} {req : Spec.Request} {entities : Spec.Entities}
+  (wf : req.WellFormed) :
+  Set.make ((Partial.isAuthorized req entities policies).errors.map Prod.fst) =
+  Spec.errorPolicies policies req entities
```

Comment:
Nit: Another naming inconsistency: `errors` vs `errorPolicies`

Reply:
Currently, `errors` returns actual errors while `errorPolicies` returns only policy IDs.  We could change this, but this has more ramifications, see thread above

Reply:
Spec is also inconsistent with `errorPolicies` vs `erroringPolicies` (perhaps they wanted to intentionally avoid the naming clash between the standalone function and the field of `Response`)?  I think changing Spec is out of scope for this PR